templates/base.html

```diff
@@ -95,6 +98,18 @@
                   <a class="nav-link" href="/{{href}}">{{id}}</a>
               </li>
               {% endfor %}
+              {% if organization_bar %}
+              <li class="nav-item dropdown {% if  active_page == 'Organization' %}active{% endif %}">
```

Comment:
Will need to go into comittee.html, sponsor.html and call-for-papers.html and update the active page in these files to match 'Organization' as set here otherwise the nav won't highlight when viewing the sub-pages

Reply:
Would it work to just grab values from `organization_bar` and check to see if the pages is in there?

Reply:
Tried adding `organization_bar_values` in the fix.

Reply:
settings active page is already built in to mini-conf, rather than adding organization_bar_values , if you open sponsors.html for example you will see the active page being set on the first line, we just need to change those to Organization